### ISSUE TITLE
Add a way to disable tooltips

### ISSUE THREAD

**Issue by DanKaplanSES** [adiessl/vscode-backtix#5]
TL;DR: I'd like to be able to turn off the tooltips.

This tool is great and does what I want, but it puts a "..." underneath the first character of _every_ string in my code. Hovering over it pops up a tooltip, but I'm not sure how that helps because it does not have any quickfix options, as you can see below:

![image](https://user-images.githubusercontent.com/373675/148667213-ae2dee0b-ccef-4a92-92a0-c816b9da2fa4.png)

I don't want to see the "..." or the tooltips because they distract me and obscure my view of the code, respectively. It seems like settings exist for changing the text of them, but I haven't been able to figure out how to turn them off:

```json
"backtix.conversionTexts": {
  "convertToSingleQuotes": "Convert to single quotes",
  "convertToDoubleQuotes": "Convert to double quotes",
  "convertToBackticks": "Convert to backticks"
}
```

I skimmed https://github.com/adiessl/vscode-backtix/blob/master/src/core/backtixCodeActionProvider.ts but nothing jumped out at me.

**Comment by adiessl** [comment]
The reason why you see the message _No quick fixes available_ is probably because BackTix uses `DiagnosticSeverity.Hint` internally:
https://github.com/adiessl/vscode-backtix/blob/129628de785bb720ec4419eb88a51ef266c55b28/src/core/utils/diagnostic.utils.ts#L14

If you click on the light bulb or use `Ctrl` / `Cmd`+ `.` the quick fixes should appear.

-----------------

I don't think it is possible for an extension to disable the tooltip behavior. You can, however, disable tooltips globally by specifying `editor.hover.enabled: false` in the `settings.json`, see https://stackoverflow.com/questions/41115285/disable-tooltip-hint-in-visual-studio-code.

If you do not want to disable the tooltips it might at least help a bit to turn off either the conversion to single or double quotes (if you only use one of them of course). Please refer to the settings section of the README.md: https://github.com/adiessl/vscode-backtix#extension-settings

I hope this helps you at least a bit. Thanks for using BackTix!
